src/data/orthoWV02_12FEB032148252-M1BS-1030010011973A00_u16ns3031_rad_atmcorr1.txt

```diff
@@ -0,0 +1,160 @@
+ENVI ASCII Plot File [Thu Oct 25 12:42:53 2018]
```

Comment:
May I ask the purpose of this file?

Reply:
In the absence of automated region of interest (ROI) selection, this is the output from the manual process.  It is needed as input to atmcorr_regr.py.

Reply:
Would it make sense then to add it in a python file and import it? That way when this code becomes part of a package, we are sure that it gets installed correctly.

Reply:
I don't think so.  It is a dynamic file.  The file will be produced by the ROI code and written to the path where atmcorr_regr.py expects to find it.  There will be a different file for each image, this is just an example. @helen-eifert Go ahead and push any others that you have for the other test images.

Reply:
Okay, I see!

Reply:
@bspitzbart do you mean all the other manual output files? Would you like me to put them in the google drive or somewhere else?